spec/index.html

```diff
@@ -722,19 +723,25 @@ <h3>Properties of simple entailment and satisfiability</h3>
     
     <p>We define the <dfn>set of propositions</dfn> in an interpretation as follows:</p>
 
-	<p class="fact"> The set of propositions in an interpretation I is IPR(I) = {&nbsp;IT(x, y, z)&#65372;x is in IR, y is in IP, z is in IR&nbsp;}; we observe that a proposition is in the extension of <code>rdfs:Proposition</code>. </p>
+	<p class="fact"> The set of propositions in an interpretation I is IPR(I) = {&nbsp;IT(x, y, z)&#65372;x is in IR, y is in IP, z is in IR&nbsp;}. </p>
+	
+	<p>The denotation of a triple is a proposition, regardless of whether it is used as a triple term or an asserted triple. Given the semantics of <a href="#rdfs_interpretations">RDFS Interpretations</a>, a proposition is in the extension of the class <code>rdfs:Proposition</code>.</p>
```

Comment:
Applying @doerthe's [suggestion](https://github.com/w3c/rdf-semantics/pull/156/files#r2322556623)...
```suggestion
	<p>The denotation of a triple is a proposition, whether it is used as a triple
	term or an asserted triple. Under <a href="#rdfs_interpretations">RDFS
	Interpretations</a> (see below), a proposition is in the extension of the
	class <code>rdfs:Proposition</code>.</p>
```